{"failedTests":[{"testName":"parallel/test-async-context-frame","timestamp":1725159499090,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 2,win2022-COMPILED_BY-vs2022 #29806","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=2,nodes=win2022-COMPILED_BY-vs2022/29806/","buildNumber":29806,"builtOn":"test-rackspace-win2022_vs2022-x64-3","nodeVersion":"23.0.0","commitHash":"68fc01d1c35d138cb0967da6e10bb0d856fefad0","callStack":[{"upstreamBuild":65363,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73762,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61769,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54591"}],"tap":{"ok":false,"name":"parallel/test-async-context-frame","id":123,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"▶ AsyncContextFrame\n  ✔ async-hooks\\test-async-local-storage-args.js (4285.5839ms)\n  ✖ async-hooks\\test-async-local-storage-async-await.js (5339.7066ms)\n    AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-await.js failed with exit code 3221225477\n        at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n        at async Test.run (node:internal/test_runner/test:888:9)\n        at async Promise.all (index 1)\n        at async Suite.run (node:internal/test_runner/test:1268:7)\n        at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n      generatedMessage: false,\n      code: 'ERR_ASSERTION',\n      actual: 3221225477,\n      expected: 0,\n      operator: 'strictEqual'\n    }\n\n  ✔ async-hooks\\test-async-local-storage-async-functions.js (5719.9215ms)\n  ✔ async-hooks\\test-async-local-storage-dgram.js (5448.9027ms)\n  ✔ async-hooks\\test-async-local-storage-enable-disable.js (5781.0054ms)\n  ✔ async-hooks\\test-async-local-storage-enter-with.js (4589.8623ms)\n  ✔ async-hooks\\test-async-local-storage-errors.js (4729.7123ms)\n  ✔ async-hooks\\test-async-local-storage-gcable.js (6023.5611ms)\n  ✔ async-hooks\\test-async-local-storage-http-agent.js (4934.4289ms)\n  ✔ async-hooks\\test-async-local-storage-http.js (5645.2999ms)\n  ✔ async-hooks\\test-async-local-storage-misc-stores.js (5646.2015ms)\n  ✔ async-hooks\\test-async-local-storage-nested.js (5422.1869ms)\n  ✔ async-hooks\\test-async-local-storage-no-mix-contexts.js (6517.606ms)\n  ✔ async-hooks\\test-async-local-storage-promises.js (5287.6545ms)\n  ✔ async-hooks\\test-async-local-storage-socket.js (5124.0097ms)\n  ✔ async-hooks\\test-async-local-storage-thenable.js (5384.8195ms)\n  ✔ async-hooks\\test-async-local-storage-tlssocket.js (5405.6258ms)\n  ✔ parallel\\test-async-local-storage-bind.js (5930.516ms)\n  ✔ parallel\\test-async-local-storage-contexts.js (5354.8162ms)\n  ✔ parallel\\test-async-local-storage-deep-stack.js (4958.255ms)\n  ✔ parallel\\test-async-local-storage-exit-does-not-leak.js (4850.987ms)\n  ✔ parallel\\test-async-local-storage-http-multiclients.js (5882.4955ms)\n  ✔ parallel\\test-async-local-storage-snapshot.js (5303.6502ms)\n▶ AsyncContextFrame (6683.1955ms)\nℹ tests 23\nℹ suites 1\nℹ pass 22\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 0\nℹ duration_ms 6698.575\n\n✖ failing tests:\n\ntest at test\\parallel\\test-async-context-frame.mjs:48:5\n✖ async-hooks\\test-async-local-storage-async-await.js (5339.7066ms)\n  AssertionError [ERR_ASSERTION]: Test async-hooks\\test-async-local-storage-async-await.js failed with exit code 3221225477\n      at TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-async-context-frame.mjs:58:7)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async Test.run (node:internal/test_runner/test:888:9)\n      at async Promise.all (index 1)\n      at async Suite.run (node:internal/test_runner/test:1268:7)\n      at async startSubtestAfterBootstrap (node:internal/test_runner/harness:280:3) {\n    generatedMessage: false,\n    code: 'ERR_ASSERTION',\n    actual: 3221225477,\n    expected: 0,\n    operator: 'strictEqual'\n  }"},"time":7200.243,"fullname":"parallel/test-async-context-frame","closingTestPoint":false}},{"testName":"parallel/test-cluster-fork-stdio","timestamp":1725159499090,"status":"REGRESSION","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 1,win2019-COMPILED_BY-vs2022 #29806","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=1,nodes=win2019-COMPILED_BY-vs2022/29806/","buildNumber":29806,"builtOn":"test-rackspace-win2019_vs2019-x64-1","nodeVersion":"23.0.0","commitHash":"68fc01d1c35d138cb0967da6e10bb0d856fefad0","callStack":[{"upstreamBuild":65363,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":73762,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":61769,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"54591"}],"tap":{"ok":false,"name":"parallel/test-cluster-fork-stdio","id":962,"buffered":false,"tapError":null,"skip":false,"todo":false,"previous":null,"plan":null,"diag":{"severity":"fail","exitcode":1,"stack":"timeout"},"time":120169.817,"fullname":"parallel/test-cluster-fork-stdio","closingTestPoint":false}}]}